committee-steering/governance/sig-governance.md

```diff
@@ -169,6 +170,30 @@ curation from other SIG participants
   - Defined in `SECURITY_CONTACTS` files, this is only relevant to the root file
    in the repository. Template [SECURITY_CONTACTS]
 
+### Other Roles
+This governance document outlines the required roles for SIGs: Chair and Tech
+Lead; however, SIGs are allowed to operate how they see fit outside of minimum 
+governance requirements, including defining more roles to sustain the group. If 
+a SIG needs to change the Chair and Tech Lead position to include or remove
+duties, this needs to be aprpoved by the Steering Committee. Newly created roles
+that don't assume any responsibilty of Chair and/or Tech Lead should follow
+the governing processes in the SIGs charter. 
+
+Example of SIG roles created to help operations:
+
+- [The Release Team: Bug Triage, CI Signal, and more]  
+- [API Reviewer and Moderator]   
+- [Production Readiness Reviewer]  
+- [Events Lead]  
+- [PR Wrangler] 
+- [Editor]
+
+Other roles...
+- *MUST* be tracked on the SIGs README with a link to the role definition
+- *MAY* Fill in for another named role on a temporary basis 
+- *MUST* have the Steering Committees approval to proceed with roles that assume
```

Comment:
move the `MUST`'s to the top? LGTM

Reply:
+1 with dims suggestion to move the musts to the top 👍